dev-tools/packaging/templates/docker/Dockerfile.elastic-agent.tmpl

```diff
@@ -53,6 +53,7 @@ RUN true && \
     true
 
 # Keep this after any chown command, chown resets any applied capabilities
+RUN setcap =p {{ $beatHome }}/data/elastic-agent-{{ commit_short }}/elastic-agent
```

Comment:
it is not about the need of resetting any capabilities, this binary to begin with doesn't have any capabilities so none is reset. To be frank, I went with `=p` (resetting) and not `all+p` (appending) because we don't want any capability to be left in the `effective` set, as this defeats the support of dropping ALL capabilities. PS: when a binary has a capability set in it;s `effective` set but the capability can't be granted, then binary doesn's start and the OS returns permission denied. Hopefully that explanation helps?! 🙂